### PR TITLE
Fix some issues with config loading...

### PR DESCRIPTION
**Pull request by brendandburns**
@arschles this fixes config loading for credential based configurations.

### CODE REVIEW

**Review by arschles — CHANGES_REQUESTED** [review]
I forgot to submit my review with "Request changes" selected. Only one [comment](https://github.com/kubernetes-client/typescript/pull/10/files#r125941845), and then good to go. Ping me when you're ready for a re-review

File: config.ts [hunk 1]
```diff
@@ -58,7 +58,10 @@ export class KubeConfig {
     private findObject(list: Object[], name: string, key: string) {
         for (let obj of list) {
             if (obj['name'] == name) {
-                return obj[key];
+                if (obj[key]) {
+                    return obj[key];
+                }
+                return obj;
             }
         }
```
Comment (brendandburns) [inline config.ts]: done.

**Comment by brendandburns** [comment]
@arschles comments addressed, please re-check.

Thanks!

**Review by arschles — APPROVED** [review]
LGTM, thanks @brendandburns

**Pull request merged**